frontend/src/app/settings/page.tsx

```diff
@@ -2,6 +2,8 @@ import React from 'react'
 
 export default function SettingsPage() {
   return (
-    <div className="main-container">Settings</div>
+    <>
```

Comment:
Nah this shouldn't be needed here. You only have one div as the page result itself. If you have multiple tags together, then you need to wrap them like this.

Reply:
I was testing another component, the toast messages, and forgot to remove the fragment when I was done. I'll make sure this doesn't happen again. It's now fixed.